source/install/use-kubernetes-mattermost.rst

```diff
@@ -0,0 +1,89 @@
+.. _use-kubernetes-mattermost:
+
+Using Mattermost Operator Functionality
+======================================
+
+Rolling Upgrades
+----------------
+
+The Mattermost Kubernetes Operator supports rolling upgrades so you can upgrade
+your Mattermost deployment with zero downtime. This process
+requires at least two replicas as a rolling upgrade cannot be performed if there is only one pod.
+Replicas are created when a user count is selected and exceeds 100.
+
+New releases are announced via our community server, as well as social media.
```

Comment:
This line seems out of place to me. Is this for the MM operator itself or for the Rolling upgrade feature?